src/rpc/rpcHandler.ts

```diff
@@ -592,31 +595,39 @@ export class RpcHandler implements IRpcEndpoint {
             return null
         }
 
-        let op: UserOperationV06 | PackedUserOperation | undefined = undefined
+        let op: UserOperationV06 | UserOperationV07
         try {
             const decoded = decodeFunctionData({
-                abi: EntryPointV06Abi,
+                abi: [...EntryPointV06Abi, ...EntryPointV07Abi],
                 data: tx.input
             })
+
             if (decoded.functionName !== "handleOps") {
                 return null
             }
+
             const ops = decoded.args[0]
-            op = ops.find(
+            const foundOp = ops.find(
                 (op: UserOperationV06 | PackedUserOperation) =>
                     op.sender === userOperationEvent.args.sender &&
                     op.nonce === userOperationEvent.args.nonce
             )
-        } catch {
-            return null
-        }
 
-        if (op === undefined) {
+            if (foundOp === undefined) {
+                return null
+            }
+
+            if (slice(tx.input, 0, 4) === "0x765e827f") {
```

Comment:
What is this selector 0x765e827f? Can we create a constant and name it?

Reply:
Good point, this magic constant/value is now removed